Makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	docker compose up --build
+	docker compose build
```

Comment:
for me the issues were with Docker CLI v2 somehow not working, I just changed the commands to old style

`docker-compose` and it's working now

Reply:
Figure we should change it? It doesn't give me any issues with it like this.